cmd/kubelet/app/server.go

```diff
@@ -1020,3 +1027,11 @@ func parseResourceList(m utilconfig.ConfigurationMap) (api.ResourceList, error)
 	}
 	return rl, nil
 }
+
+func parseKernelTunableBehaviour(protect bool) cm.KernelTunableBehavior {
```

Comment:
Since there are three behaviours. At the moment, the behaviour flag is boolean. Later, it can be changed/removed/extended with more flags. In order to make the transition from input to any of the three (and in future more) behaviour generic, I kept the parse function here.